app/templates/main/map.html

```diff
@@ -0,0 +1,100 @@
+                // Use latitude and longitude values from incident report
+                // to set position of marker
+                position:{lat: parseFloat('{{ report.location.latitude }}'), lng: parseFloat('{{ report.location.longitude }}')}
+            });
+            var dateString = '{{ report.date }}';
+            year = parseInt((dateString.split(' '))[0].split('-')[0]);
+            month = parseInt((dateString.split(' '))[0].split('-')[1])-1;
+            day = parseInt((dateString.split(' '))[0].split('-')[2]);
+            incidentDate = new Date(year, month, day);
+            if (incidentDate.getTime() < minDate.getTime())
+                minDate.setTime(incidentDate.getTime());
+            //marker.setMap(map);
+            var contentString = '<div id="content">' +
+                                '<div id="siteNotice">' +
+                                '</div>' +
+                                '<h1 id="firstHeading" class="firstHeading">Vehicle ID: ' + '{{ report.vehicle_id }}' +'</h1>' +
+                                '<div id="bodyContent">' +
+                                '<p>License Plate: ' + '{{ report.license_plate }}' + '</p>' +
+                                '<p>Date: ' + '{{ report.date }}' + '</p>' +
+                                '<p>Duration: ' + '{{ report.duration }}' + '</p>' +
+                                '<p>Agency: ' + '{{ report.agency.name }}' + '</p>' +
+                                '<p>Link to Picture: ' + '{{ report.picture_url }}' + '</p>' +
+                                '<p>Description: ' + '{{ report.description }}' + '</p>' +
+                                '</div>' +
+                                '</div>';
+            bounds.extend(marker.getPosition());
+            markerWrapper = new MarkerWrapper(marker, incidentDate, contentString);
+            markerWrappers.push(markerWrapper);
+            var infoWindow = new google.maps.InfoWindow({
+                content: contentString
+            });
+
+            // Add click listener to marker for displaying infoWindow
+            marker.addListener('click', function() {
+                infoWindow.open(map, marker);
+            });
+        });
+        {% endfor %}
+        var mapProp = {
+                mapTypeId: google.maps.MapTypeId.ROADMAP,
+                mapTypeControl: false,
+                streetViewControl: false,
+                scrollwheel: false,
+        };
+        map=new google.maps.Map($('#googleMap')[0],mapProp);
+
+        storeMarkerState(markerWrappers, map, minDate, bounds);
+        initializeDateSlider();
+        addLocationButton(map);
+    }
+    $(document).ready(function() {
+        // Add the map to the window
+        google.maps.event.addDomListener(window, 'load', initializeMapAndMarkers);
+
+        // Set the height of the googleMap to the height of the window (less
+        // any elements above the googleMap)
+        $(function () {
```

Comment:
You might want to change the margin top of the footer here so that we don't have white space between the map and the footer